testing/src/main/java/io/grpc/internal/testing/AbstractTransportTest.java

```diff
@@ -177,6 +178,22 @@ public void setUp() {
     when(serverStreamTracerFactory.newServerStreamTracer(anyString(), any(Metadata.class)))
         .thenReturn(serverStreamTracer);
     callOptions = CallOptions.DEFAULT.withStreamTracerFactory(clientStreamTracerFactory);
+
+    doAnswer(
```

Comment:
Eww... each test needing to do something different is pretty ugly. It could be made better if you made the Answers non-anonymous. But I do wonder if there is something better we can do. Maybe something with Captors... Dunno.